pyccel/parser/semantic.py

```diff
@@ -1132,6 +1136,74 @@ def _handle_function(self, expr, func, args, *, is_method = False, use_build_fun
                             func, func.is_elemental)
 
             return new_expr
+    def _handle_kernel(self, expr, func, args, **settings):
+        """
+        Create the node representing the kernel function call.
+
+        Create a FunctionCall or an instance of a PyccelInternalFunction
+        from the function information and arguments.
+
+        Parameters
+        ----------
+        expr : PyccelAstNode
+               The expression where this call is found (used for error output).\
+
+        func : FunctionDef instance, Interface instance or PyccelInternalFunction type
+               The function being called.
+
+        args : tuple
+               The arguments passed to the function.
+
+        **settings : dict
+            The settings passed to _visit functions.
+
+        Returns
+        -------
+        FunctionCall/PyccelInternalFunction
+            The semantic representation of the call.
+        """
+        if(len(expr.launch_config) < 2):
```

Comment:
Not every `PyccelAstNode` has a `launch_config` property. Are you sure that this describes the argument type accurately?

Reply:
sorry emily i didn't see that u right as always fixed fa92012

Reply:
i pass the IndexedFunctionCall that always has  launch_config but i made mistake in doc string